### PR TITLE
feat: DW-414 add buy a plan from webapp link

### PR DESCRIPTION
**Pull request by cbernat**
### Change plan basic test
Add simple page to test buy a plan from webapp connected with old Doppler, also add a link to this section in new features page,

image

I´ve worked with a dynamic route so we have this possibilities to test in near future plus plans.
- https://cdn.fromdoppler.com/doppler-webapp/demo-build2550/#/plans/18/discounts/2/promotions/ALLPLANS/buy (access plan 18 with the discount and promo code)
- https://cdn.fromdoppler.com/doppler-webapp/demo-build2550/#/plans/18/promotions/ALLPLANS/discounts/2/buy
- https://cdn.fromdoppler.com/doppler-webapp/demo-build2550/#/plans/18/buy
- https://cdn.fromdoppler.com/doppler-webapp/demo-build2550/#/plans/18/discounts/2/buy
- https://cdn.fromdoppler.com/doppler-webapp/demo-build2550/#/plans/18/promotions/ALLPLANS/buy

cdn link for int
https://cdn.fromdoppler.com/doppler-webapp/int-build2550/#/plans/18/discounts/2/promotions/ALLPLANS/buy

This way we can test any plan. Even a plus plan when we have it.

### CODE REVIEW

**Comment by fcoronelmakingsense** [comment]
💪

File: src/components/ChangePlan/ChangePlan.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import { useRouteMatch } from 'react-router-dom';
+import { Helmet } from 'react-helmet';
+
+const ChangePlan = () => {
+  const { params } = useRouteMatch();
+  const { planId, discountId, promoId } = params;
+  const safeDiscountId = discountId ? discountId : 0;
```
Comment (lucianaDilizio) [inline src/components/ChangePlan/ChangePlan.js:9]: Why not add a safePlanId? 🤔

Comment (cbernat) [reply src/components/ChangePlan/ChangePlan.js:9]: The plan is a required parameter, must be there. The other ones are optionals.

**Comment by javierburger** [comment]
👍

**Pull request merged**